docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -75,21 +81,21 @@ Common Arguments for all Execution Policies
+| Argument          | Options                                                                   | Purpose                                                                                                        | Default                    |
++===================+===========================================================================+================================================================================================================+============================+
+| ExecutionSpace    | ``Serial``, ``OpenMP``, ``Threads``, ``Cuda``, ``HIP``, ``SYCL``, ``HPX`` | Specify the Execution Space to execute the kernel in.                                                          | ``DefaultExecutionSpace``  |
++-------------------+---------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+----------------------------+
+| Schedule          | ``Schedule<Dynamic>``, ``Schedule<Static>``                               | Specify scheduling policy for work items. ``Dynamic`` scheduling is implemented through a work stealing queue. | machine / backend specific |
++-------------------+---------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+----------------------------+
+| IndexType         | ``IndexType<IT>``                                                         | Specify integer type to be used for traversing the iteration space.                                            | ``IndexType<int64_t>``     |
++-------------------+---------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+----------------------------+
+| LaunchBounds      | ``LaunchBounds<MaxThreads, MinBlocks>``                                   | Specifies hints to to the compiler about CUDA/HIP launch bounds.                                               |                            |
++-------------------+---------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+----------------------------+
+| WorkTag           | ``SomeClass``                                                             | Specify the work tag type used to call the functor operator.                                                   | ``void``                   |
++-------------------+---------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+----------------------------+
 
 Public Class Members
```

Comment:
The "Public Class Members" section should be moved to the "Interface" section to conform with the new API entry standard.